tsconfig.base.json

```diff
@@ -5,14 +5,14 @@
     "esModuleInterop": true,
     "jsx": "react-jsx",
     "lib": ["dom", "dom.iterable", "esnext"],
-    "module": "esnext",
+    "module": "ESNext",
     "moduleResolution": "node",
     "noImplicitAny": true,
     "noUnusedLocals": true,
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
-    "target": "es2020",
+    "target": "ESNext",
```

Comment:
I believe we need to set this to a specific version for compatibility reasons.
es2020 might make sense. We can upgrade it manually to new targets with version increase but esnext seems to cause compatibility issues since it can be any thing at the compile/publish time of the lib

https://www.typescriptlang.org/tsconfig#target

The special ESNext value refers to the highest version your version of TypeScript supports. This setting should be used with caution, since it doesn’t mean the same thing between different TypeScript versions and can make upgrades less predictable